annotations/admin.py

```diff
@@ -36,10 +36,21 @@ class VogonUserAdmin(UserAdmin):
 class TextAdmin(admin.ModelAdmin):
     list_display = ('uri', 'title', 'created')
 
+class RelationAdmin(admin.ModelAdmin):
```

Comment:
Oh, and a styling thing: always leave two blank lines above a top-level class or function declaration. See [this section in PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines).